external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -965,6 +965,92 @@ impl std::fmt::Display for Value_ {
     }
 }
 
+//**************************************************************************************************
+// Shared
+//**************************************************************************************************
+
+// Implicit aliases for the Move Stdlib:
+// use std::vector;
+// use std::option::{Self, Option};
+pub const IMPLICIT_STD_MODULES: &[Symbol] = &[symbol!("option"), symbol!("vector")];
+pub const IMPLICIT_STD_MEMBERS: &[(Symbol, Symbol, ModuleMemberKind)] = &[(
+    symbol!("option"),
+    symbol!("Option"),
+    ModuleMemberKind::Struct,
+)];
+
+// Implicit aliases for Sui mode:
+// use sui::object::{Self, ID, UID};
+// use sui::transfer;
+// use sui::tx_context::{Self, TxContext};
+pub const IMPLICIT_SUI_MODULES: &[Symbol] = &[
+    symbol!("object"),
+    symbol!("transfer"),
+    symbol!("tx_context"),
+];
+pub const IMPLICIT_SUI_MEMBERS: &[(Symbol, Symbol, ModuleMemberKind)] = &[
+    (symbol!("object"), symbol!("ID"), ModuleMemberKind::Struct),
+    (symbol!("object"), symbol!("UID"), ModuleMemberKind::Struct),
+    (
+        symbol!("tx_context"),
+        symbol!("TxContext"),
+        ModuleMemberKind::Struct,
+    ),
+];
```

Comment:
Consider `implicit_imports.rs` or similar for these definitions, because they don't really belong in the AST file.

Reply:
Yeah, a strange one.
We currently have `Flavor` defined in `editions/mod.rs` but maybe we need a `flavor` module. I think default implementations would make a lot of sense in that module, since the default imports are defined by the flavor

Reply:
Not sure if there is a concrete suggestion here on where to move it in this PR or it's something we are thinking about but could wait for a separate PR...